### DISCUSSION TITLE
Adding Rec2020 Limited to P3 Target Colour Space

### DISCUSSION THREAD

**Discussion by JalfasV2** [Netflix/OpenVPCal#61] [Ideas]
Hello, hope you are well...

So I have had discussions where the question is: Should the LED be set to Rec2020 or to P3? 

There are pros and cons but I think is would be very useful if OpenVPCal added the Rec2020 P3 Limited as a Gamut target option. My wall, for example, is set to 2020. When I tried to set everything to P3 the results where not great. I believe that this was because I calibrated with a CR300 colorimeter to 2020, therefore, my processors are optimized to these colour primaries instead of the ones of P3. I know that a solution is to calibrate the wall to P3 but this would be very expensive. A quick and good solution would be to use these same primaries and to limit the gamut to P3. This is why it would be amazing to add this feature to OpenVPCal.

Thank you for your time and understanding, 
Sincerely, Juan Fernando

**Comment by davisadam10** [comment]
If you follow the custom gamut workflow you can target the exact capabilities of the led wall itself.

For the most precise calibrations adding in a custom gamut based on the xy values for the rgb primaries and the whitepoint will give you the highest possible utilisation of the imaging chain.

Assuming your processor supports utilising the achievable gamut of the led wall
